docs/topics/monitoring.md

```diff
@@ -54,6 +54,29 @@ NAME            CPU(cores)   MEMORY(bytes)
 somePod         0m           11Mi
 ```
 
+## Azure Monitor for containers
+
+Azure Monitor for containers is a feature designed to monitor the performance of container workloads deployed to AKS-engine (formerly known as ACS-engine) cluster(s) hosted on Azure. Monitoring your containers is critical, especially when you're running a production cluster, at scale, with multiple applications.
```

Comment:
let's standardize the spelling of `AKS-Engine` everywhere in this doc

Reply:
(same for ACS-Engine)

Reply:
@CecileRobertMichon, I see the name is not consistent across the docs. What is the standardize name we should update for AKS Engine and ACS engine in our docs? should we go with AKS-Engine and ACS-Engine?

Reply:
correct, we're trying to get to a consistent state and converge to `AKS-Engine` and `ACS-Engine` (same as they appear in the README)

Reply:
updated as discussed all the changes. @CecileRobertMichon, please have a look at them.